lms/static/sass/discussion/_discussion.scss

```diff
@@ -625,7 +563,7 @@ body.discussion {
     width: 100%;
     margin-bottom: $baseline;
     @include clearfix();
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
I think the `@include` uses Bourbon and automatically spits out different rules for various browsers.